### ISSUE TITLE
Header Stringtable Key Stack Overflow

### ISSUE THREAD

**Issue by kayler-renslow** [kayler-renslow/arma-intellij-plugin#40]
### Plugin Version
1.0.7
### Summary
StackOverFlowException when stringtable key is used
### Detail
This happens often.

```
null
java.lang.StackOverflowError
	at com.intellij.openapi.roots.impl.RootIndex.getInfoForFile(RootIndex.java:400)
	at com.intellij.openapi.roots.impl.DirectoryIndexImpl.getInfoForFile(DirectoryIndexImpl.java:143)
	at com.intellij.openapi.roots.impl.FileIndexBase.getInfoForFileOrDirectory(FileIndexBase.java:29)
	at com.intellij.openapi.roots.impl.ProjectFileIndexImpl.getModuleForFile(ProjectFileIndexImpl.java:118)
	at com.intellij.openapi.roots.impl.ProjectFileIndexImpl.getModuleForFile(ProjectFileIndexImpl.java:111)
	at com.kaylerrenslow.a3plugin.util.PluginUtil.getModuleForPsiFile(PluginUtil.java:119)
	at com.kaylerrenslow.a3plugin.lang.header.psi.mixin.HeaderStringtableKeyMixin.getReferences(HeaderStringtableKeyMixin.java:42)
	at com.kaylerrenslow.a3plugin.lang.header.psi.mixin.HeaderStringtableKeyMixin.getReference(HeaderStringtableKeyMixin.java:36)
	at com.intellij.psi.impl.SharedPsiElementImplUtil.getReferences(SharedPsiElementImplUtil.java:93)
	at com.intellij.psi.impl.PsiElementBase.getReferences(PsiElementBase.java:82)
	at com.kaylerrenslow.a3plugin.lang.header.psi.mixin.HeaderStringtableKeyMixin.getReferences(HeaderStringtableKeyMixin.java:59)
	at com.kaylerrenslow.a3plugin.lang.header.psi.mixin.HeaderStringtableKeyMixin.getReference(HeaderStringtableKeyMixin.java:36)
	at com.intellij.psi.impl.SharedPsiElementImplUtil.getReferences(SharedPsiElementImplUtil.java:93)
	at com.intellij.psi.impl.PsiElementBase.getReferences(PsiElementBase.java:82)
	at com.kaylerrenslow.a3plugin.lang.header.psi.mixin.HeaderStringtableKeyMixin.getReferences(HeaderStringtableKeyMixin.java:59)
	at com.kaylerrenslow.a3plugin.lang.header.psi.mixin.HeaderStringtableKeyMixin.getReference(HeaderStringtableKeyMixin.java:36)
	at com.intellij.psi.impl.SharedPsiElementImplUtil.getReferences(SharedPsiElementImplUtil.java:93)
	at com.intellij.psi.impl.PsiElementBase.getReferences(PsiElementBase.java:82)
	at com.kaylerrenslow.a3plugin.lang.header.psi.mixin.HeaderStringtableKeyMixin.getReferences(HeaderStringtableKeyMixin.java:59)
	at com.kaylerrenslow.a3plugin.lang.header.psi.mixin.HeaderStringtableKeyMixin.getReference(HeaderStringtableKeyMixin.java:36)
	at com.intellij.psi.impl.SharedPsiElementImplUtil.getReferences(SharedPsiElementImplUtil.java:93)
	at com.intellij.psi.impl.PsiElementBase.getReferences(PsiElementBase.java:82)
	at com.kaylerrenslow.a3plugin.lang.header.psi.mixin.HeaderStringtableKeyMixin.getReferences(HeaderStringtableKeyMixin.java:59)
	at com.kaylerrenslow.a3plugin.lang.header.psi.mixin.HeaderStringtableKeyMixin.getReference(HeaderStringtableKeyMixin.java:36)
	at com.intellij.psi.impl.SharedPsiElementImplUtil.getReferences(SharedPsiElementImplUtil.java:93)
	at com.intellij.psi.impl.PsiElementBase.getReferences(PsiElementBase.java:82)
	at com.kaylerrenslow.a3plugin.lang.header.psi.mixin.HeaderStringtableKeyMixin.getReferences(HeaderStringtableKeyMixin.java:59)
	at com.kaylerrenslow.a3plugin.lang.header.psi.mixin.HeaderStringtableKeyMixin.getReference(HeaderStringtableKeyMixin.java:36)
	at com.intellij.psi.impl.SharedPsiElementImplUtil.getReferences(SharedPsiElementImplUtil.java:93)
	at com.intellij.psi.impl.PsiElementBase.getReferences(PsiElementBase.java:82)
	at com.kaylerrenslow.a3plugin.lang.header.psi.mixin.HeaderStringtableKeyMixin.getReferences(HeaderStringtableKeyMixin.java:59)
	at com.kaylerrenslow.a3plugin.lang.header.psi.mixin.HeaderStringtableKeyMixin.getReference(HeaderStringtableKeyMixin.java:36)
	at com.intellij.psi.impl.SharedPsiElementImplUtil.getReferences(SharedPsiElementImplUtil.java:93)
	at com.intellij.psi.impl.PsiElementBase.getReferences(PsiElementBase.java:82)
	at com.kaylerrenslow.a3plugin.lang.header.psi.mixin.HeaderStringtableKeyMixin.getReferences(HeaderStringtableKeyMixin.java:59)
	at com.kaylerrenslow.a3plugin.lang.header.psi.mixin.HeaderStringtableKeyMixin.getReference(HeaderStringtableKeyMixin.java:36)
	at com.intellij.psi.impl.SharedPsiElementImplUtil.getReferences(SharedPsiElementImplUtil.java:93)
	at com.intellij.psi.impl.PsiElementBase.getReferences(PsiElementBase.java:82)
	at com.kaylerrenslow.a3plugin.lang.header.psi.mixin.HeaderStringtableKeyMixin.getReferences(HeaderStringtableKeyMixin.java:59)
	at com.kaylerrenslow.a3plugin.lang.header.psi.mixin.HeaderStringtableKeyMixin.getReference(HeaderStringtableKeyMixin.java:36)
	at com.intellij.psi.impl.SharedPsiElementImplUtil.getReferences(SharedPsiElementImplUtil.java:93)
	at com.intellij.psi.impl.PsiElementBase.getReferences(PsiElementBase.java:82)

```
#### Reproduce code: 
```
class EquipmentModuleSpacer{
	title = "";
	values[] = { "" };
	texts[] = { "" };
	default = "";
};

class EquipmentModuleOptionsTitle{
	title = $STR_PARAMS_FAR_REVIVE_OPTIONS;
	values[] = { "" };
	texts[] = { "" };
	default = "";
};

class EquipmentModuleUseRandomAiLoadouts{
	title = $STR_PARAMS_USE_RANDOM_UI_LOADOUT;
	values[] = {1,0};
	texts[] = { $STR_OPTION_YES, $STR_OPTION_NO };
	default = 1;
};
```

**Comment by kayler-renslow** [comment]
Fixed in commit [d479c5a](https://github.com/kayler-renslow/arma-intellij-plugin/commit/d479c5a72f051268280328c44e140fcc3609501d)

**Comment by kayler-renslow** [comment]
fixed in release 1.0.8
